.github/workflows/add-docker-tag.yaml

```diff
@@ -0,0 +1,50 @@
+name: Add Tags to Blockchain Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      origin-tag:
+        description: 'Original tag'
+        required: true
+        type: string
+      destination-tag:
+        description: 'Tag to add'
+        required: true
+        type: choice
+        options:
+          - alfajores
+          - mainnet
+  workflow_call:
+    inputs:
+      origin-tag:
+        description: 'Original tag'
+        required: true
+        type: string
+      destination-tag:
+        description: 'Tag to add'
+        required: true
+        default: 'baklava'
```

Comment:
Excuse me my ignorance but why `baklava` is a special case?

Reply:
I was [following this guide](https://github.com/celo-org/celo-labs/blob/master/packages/operations/celo-blockchain-deploy-guide.md#signing-and-tagging-docker-images):

> Tag the docker images with the semantic versions (x.y.z, x.y, x) and with the network tag (baklava, alfajores, mainnet). This can be done on the GCP container registry or with the given script.
The network tags should be updated as the release is rolled out the that particular network. So the baklava tag is immediately updated, then 1 week later the alfajores tag is updated, then 1 week later the mainnet tag is updated.

So the `baklava` docker image is added when pushing a GH tag (along with the commit, the Major, the Major.Minor and the full semver). `alfajores` and `mainnet tags can be added by running the add-tag workflow manually later on (doc says 1 week between tags). For example:

![Screenshot 2023-04-21 at 09 51 24](https://user-images.githubusercontent.com/35505302/233576686-a93912d6-d30a-485d-ba8f-002a6d6723ec.png)

Happy to change if that doc is not up to date with the current release cycle.

Reply:
No need to update, I just wasn't aware of the process. Thanks for the clarification!